src/util/nextThought.js

```diff
@@ -126,6 +125,14 @@ const nextInContextView = (value, rank, path, rankedContext, contextChain, ignor
   const context = pathToContext(rankedContext)
   const firstChild = perma(() => firstChildOfContextView(path, thoughtIndex))
 
+  const contextWithoutChildren = contextChain.length === 1 &&
+  isContextViewActive(contextChain[0]) &&
+  getContexts(head(pathToContext(contextChain[0]))).length <= 1
```

Comment:
Splitting a long logical statement onto separate lines is great, just make sure to indent the second and third lines.